helpers-es5.js

```diff
@@ -187,7 +188,6 @@ var path = require('path'),
                     print('Files:create', 'Creating file: ' + name);
                     if (name === 'manifest.json') {
                         properties.name = options.appName;
-                        properties.short_name = options.appName;
```

Comment:
This file was compiled (via babel) from helpers.js and `properties.short_name` doesn't seem to exist there.